### DISCUSSION TITLE
Can't send verification email

### DISCUSSION THREAD

**Discussion by irakli2206** [pocketbase/pocketbase#1137] [Q&A] [answered]
Using the admin panel hosted on fly.io, sending test email from mail settings gives next error:

`Failed to send the test email. Raw error: exit status 1.`

I tried sending verification emails using the JS SDK and API endpoint (using NextJS), but I receive no email in both cases and no errors either (except auto-cancellation if I do it multiple times). 

Note: I did look in All Mails including Spam and tried other email addresses too.

**Comment by ganigeorgiev** [comment, marked as answer]
By default PocketBase uses the unix `sendmail` command to send emails.
The above error could be returned if:
- you are on a platform (eg. Windows or some stripped-down linux distros) that doesn't have natively the `sendmail` command
- the `sendmail` service is not started (to check its status, you could run `sudo systemctl status sendmail`)
- the `sendmail` service is not properly configured or the mail recipient directly reject the email (to inspect the sent email you could explore the sendmail logs, usually at `cat /var/spool/mail/your_user`)

In general, for sending emails I recommend using an SMTP service (mailgun, mailjet, ses, etc.).

For testing purposes I often use [mailtrap.io](https://mailtrap.io/) (they have free plan and includes email inspector and preview).
